lib/auth/legacy.js

```diff
@@ -57,6 +57,7 @@ const login = async (opts) => {
   }
 
   try {
+    log.notice('', `Log in on ${opts.registry}`)
```

Comment:
One more thing, we need to sanitize `opts.registry` output here using [`lib/utils/replace-info.js`](https://github.com/npm/cli/blob/latest/lib/utils/replace-info.js), e.g:

```diff
-    log.notice('', `Log in on ${opts.registry}`)
+    log.notice('', `Log in on ${replaceInfo(opts.registry)}`)
```